### PR TITLE
ASTRO 1118 export svg

### PR DESCRIPTION
**Pull request by Cheshire89**
create sketch plugin to export svg to `~\Documents\astro-exports\`
create node script to format svg

Presently two process are decoupled because sketch api can only be ran inside Sketch App which created poor user experience. 

Major clean up ... cleaned project up to run as true cli instead of spaghetti code

### CODE REVIEW

File: bin/generate_compact_svg/run.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      line = line.replace(originalId, newId);
+    }
+    newArr.push(line);
+  });
+
+  return newArr.join('\n');
+};
+
+const run = () => {
+  console.log('Pressing [Enter] will during promps will use default values');
```
Comment (dmcalester) [inline bin/generate_compact_svg/run.js:153]: Could you clarify this if this is an instruction to the user? If it’s not for users please remove.

I think it’s meant to ready "Pressing [ENTER] during prompts will use default values".

File: bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "script" : "script.js",
+      "name" : "export-svg",
+      "handlers" : {
+        "run" : "onRun"
+      },
+      "identifier" : "com.bohemiancoding.sketch.runscriptidentifier"
+    }
+  ],
+  "menu" : {
+    "title" : "export-svg",
```
Comment (dmcalester) [inline bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json:14]: Let’s change this to Astro Icon Export

File: bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "author" : "Aleksandr Antonov",
+  "commands" : [
+    {
+      "script" : "script.js",
+      "name" : "export-svg",
```
Comment (dmcalester) [inline bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json:6]: Let’s change this to **astro-icon-export**

File: bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "author" : "Aleksandr Antonov",
```
Comment (dmcalester) [inline bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json:2]: Please change to **Rocket Communications**

File: bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    "title" : "export-svg",
+    "items" : [
+      "com.bohemiancoding.sketch.runscriptidentifier"
+    ]
+  },
+  "identifier" : "com.example.sketch.39b943b7-516d-4d1a-b2b5-cb3bb21ee4c9",
+  "version" : "1.0",
+  "description" : "",
+  "authorEmail" : "",
+  "name" : "export-svg"
```
Comment (dmcalester) [inline bin/generate_compact_svg/export-svg.sketchplugin/Contents/Sketch/manifest.json:23]: Please rename to **astro-icon-export**

File: bin/generate_compact_svg/run.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
+/* eslint-disable */
```
Comment (dmcalester) [inline bin/generate_compact_svg/run.js:1]: Does this process run during any of the Astro build processes?

File: bin/generate_compact_svg/run.js [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+  fs.readFile(filePath, "utf-8", (err, data) => {
+    if (err) {
+      throw err;
+    }
+    
+    svgo_inst.optimize(data).then((res) => {
+      res.data = cleanUpIds(res.data);
+  
+      const savePath = __dirname + "/../../static/icons/";
+      const fileName = prompt("Please enter desired file name: ", 'astro-new.svg');
```
Comment (dmcalester) [inline bin/generate_compact_svg/run.js:165]: Let’s change the default prop to **astro.svg**

File: bin/generate_compact_svg/run.js (hunk 6 above)
Comment (Cheshire89) [reply bin/generate_compact_svg/run.js:1]: No this is so that my code formatter does not highlight node files

**Review by dmcalester — APPROVED** [review]

**Pull request merged**